modules/core/shared/src/main/scala/lucuma/core/geom/gmos/GmosCandidatesArea.scala

```diff
@@ -39,5 +40,36 @@ trait GmosCandidatesArea {
       a <- posAngles
       o <- offsetPositions
     } yield candidatesAreaAt(a, o)).fold(ShapeExpression.Empty)(_ ∪ _)
+
+  /**
+    * GMOS inner radius
+    */
+  def innerRadius(probe: GuideProbe): ShapeExpression =
+    probe match {
+      case GuideProbe.GmosOIWFS =>
+        // 20 arcsec
+        ShapeExpression.centeredEllipse((20 * 2).arcsec, (20 * 2).arcsec)
+      case GuideProbe.PWFS1 =>
+        ShapeExpression.centeredEllipse((345 * 2).arcsec, (345 * 2).arcsec)
+      case GuideProbe.PWFS2 =>
+        ShapeExpression.centeredEllipse((315 * 2).arcsec, (315 * 2).arcsec)
+    }
```

Comment:
eventually there would be F2, canopus, etc. guide probes here which i guess would need to all be handled via `case _ =>`

Reply:
Right, we'd be adding other instrument's probes and for OI too